.github/workflows/build.yml

```diff
@@ -293,4 +293,29 @@ jobs:
         run: |
           cd packages/browser
           yarn test:integration
+
+  job_browser_build_tests:
+    name: Browser Build Tests
+    needs: job_build
+    runs-on: ubuntu-latest
+    timeout-minutes: 5
```

Comment:
Using a smaller number than the other jobs because this should not take long, it is just a trivial script.
The GHA default is 360 minutes, which is too much.